src/translations/data.json

```
@@ -591,8 +591,8 @@
     "uk": "",
     "be": "",
     "pl": "Używamy plików cookie i innych technologii śledzenia, aby poprawić jakość przeglądania naszej witryny, wyświetlać spersonalizowane treści i reklamy, analizować ruch w naszej witrynie i wiedzieć, skąd pochodzą nasi użytkownicy. Przeglądając naszą stronę, wyrażasz zgodę na używanie przez nas plików cookie i innych technologii śledzenia.",
-    "cs": "",
-    "sk": "",
+    "cs": "Používáme soubory cookie a další technologie sledování, abychom zlepšili vaše prohlížení našich webových stránek, zobrazovali vám přizpůsobený obsah a cílené reklamy, analyzovali provoz na našich webových stránkách a porozuměli, odkud naši návštěvníci přicházejí. Procházením našich webových stránek souhlasíte s naším používáním souborů cookie a dalších sledovacích technologií.",
+    "sk": "Používame súbory cookie a ďalšie technológie sledovania, aby sme zlepšili vaše prezeranie našich webových stránok, zobrazovali vám prispôsobený obsah a cielené reklamy, analyzovali prevádzku na našich webových stránkach a porozumeli, odkiaľ naši návštevníci prichádzajú. Prechádzaním našich webových stránok súhlasíte s naším používaním súborov cookie a ďalších sledovacích technológií.",
```

it's probably more suitable "prehliadanie"
```suggestion
    "sk": "Používame súbory cookie a ďalšie technológie sledovania, aby sme zlepšili vaše prehliadanie našich webových stránok, zobrazovali vám prispôsobený obsah a cielené reklamy, analyzovali prevádzku na našich webových stránkach a porozumeli, odkiaľ naši návštevníci prichádzajú. Prehliadaním našich webových stránok súhlasíte s naším používaním súborov cookie a ďalších sledovacích technológií.",
```